pom.xml

```diff
@@ -67,4 +67,23 @@
     </plugins>
   </build>
 
+  <profiles>
+    <!--
+      The QA module is enabled by default, but having it in a profile allows us to exclude it easily.
+      That is useful for example for creating releases. This profile is active if the skipQaBuild
+      property is not present, or is present and set to a value other than 'true'.
+      -->
+    <profile>
+      <id>integration-tests-build</id>
+      <activation>
+        <property>
+          <name>skipQaBuild</name>
+          <value>!true</value>
```

Comment:
does it match when `-D skipQaBuild` property is used without any value ? It is used this way in many places

Reply:
Great question @houssain-barouni !

I just tried it out (on a main branch):


```
mvnd validate 

[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Camunda 8 Root 8.6.0-SNAPSHOT:
[INFO] 
[INFO] Zeebe BOM .......................................... SUCCESS [  0.000 s]
[INFO] Zeebe Build Tools .................................. SUCCESS [  0.000 s]
[INFO] Zeebe Parent ....................................... SUCCESS [  0.047 s]
[INFO] Zeebe Util ......................................... SUCCESS [  0.148 s]
[INFO] Camunda Search Parent .............................. SUCCESS [  0.145 s]
[INFO] Camunda Search Client .............................. SUCCESS [  0.088 s]
[INFO] Camunda Search Client - Elasticsearch .............. SUCCESS [  0.160 s]
[INFO] Camunda Search Client - Opensearch ................. SUCCESS [  0.072 s]
[INFO] Camunda Search Client - Connect .................... SUCCESS [  0.110 s]
[INFO] Camunda Service .................................... SUCCESS [  0.052 s]
[INFO] Zeebe Atomix Parent Pom ............................ SUCCESS [  0.096 s]
[INFO] Zeebe Atomix Utilities ............................. SUCCESS [  0.110 s]
[INFO] Zeebe Scheduler .................................... SUCCESS [  0.120 s]
[INFO] Zeebe Protocol ..................................... SUCCESS [  0.109 s]
[INFO] Zeebe Exporter API ................................. SUCCESS [  0.088 s]
[INFO] Zeebe BPMN model API ............................... SUCCESS [  0.197 s]
[INFO] Zeebe Gateway Protocol ............................. SUCCESS [  0.076 s]
[INFO] Zeebe Gateway Protocol Implementation .............. SUCCESS [  0.110 s]
[INFO] Camunda Client Java ................................ SUCCESS [  0.185 s]
[INFO] Zeebe Test Util .................................... SUCCESS [  0.081 s]
[INFO] Zeebe Snapshots .................................... SUCCESS [  0.064 s]
[INFO] Zeebe Journal ...................................... SUCCESS [  0.065 s]
[INFO] Zeebe Atomix Cluster ............................... SUCCESS [  0.170 s]
[INFO] Zeebe DB ........................................... SUCCESS [  0.110 s]
[INFO] Zeebe Msgpack Core ................................. SUCCESS [  0.044 s]
[INFO] Zeebe Msgpack Value ................................ SUCCESS [  0.083 s]
[INFO] Zeebe Auth ......................................... SUCCESS [  0.102 s]
[INFO] Zeebe Protocol Implementation ...................... SUCCESS [  0.107 s]
[INFO] Zeebe Logstreams ................................... SUCCESS [  0.074 s]
[INFO] Zeebe Stream Platform .............................. SUCCESS [  0.097 s]
[INFO] Zeebe Dynamic Cluster Configuration ................ SUCCESS [  0.076 s]
[INFO] Zeebe Transport .................................... SUCCESS [  0.090 s]
[INFO] Zeebe Protocol AssertJ Assertions .................. SUCCESS [  0.068 s]
[INFO] Zeebe Protocol Test Util ........................... SUCCESS [  0.123 s]
[INFO] Zeebe Broker Client ................................ SUCCESS [  0.102 s]
[INFO] Zeebe Gateway ...................................... SUCCESS [  0.097 s]
[INFO] Zeebe Gateway gRPC ................................. SUCCESS [  0.133 s]
[INFO] Zeebe FEEL Integration ............................. SUCCESS [  0.073 s]
[INFO] Zeebe Expression Language .......................... SUCCESS [  0.113 s]
[INFO] Zeebe DMN Engine ................................... SUCCESS [  0.108 s]
[INFO] Zeebe Workflow Engine .............................. SUCCESS [  0.183 s]
[INFO] Zeebe Backup ....................................... SUCCESS [  0.096 s]
[INFO] Zeebe Backup Store Testkit ......................... SUCCESS [  0.057 s]
[INFO] Zeebe Backup Store for S3 .......................... SUCCESS [  0.134 s]
[INFO] Zeebe Backup Store Common .......................... SUCCESS [  0.052 s]
[INFO] Zeebe Backup Store for GCS ......................... SUCCESS [  0.209 s]
[INFO] Zeebe Backup Store for Azure ....................... SUCCESS [  0.194 s]
[INFO] Zeebe Exporter Test Harness ........................ SUCCESS [  0.070 s]
[INFO] Zeebe Broker ....................................... SUCCESS [  0.087 s]
[INFO] Zeebe Backup Restore ............................... SUCCESS [  0.064 s]
[INFO] Zeebe Gateway REST API server ...................... SUCCESS [  0.117 s]
[INFO] Zeebe Protocol Jackson ............................. SUCCESS [  0.091 s]
[INFO] Zeebe Elasticsearch Exporter ....................... SUCCESS [  0.096 s]
[INFO] Zeebe Opensearch Exporter .......................... SUCCESS [  0.101 s]
[INFO] Webapps Common ..................................... SUCCESS [  0.070 s]
[INFO] Operate Parent ..................................... SUCCESS [  0.069 s]
[INFO] Operate Common ..................................... SUCCESS [  0.274 s]
[INFO] Operate schema ..................................... SUCCESS [  0.112 s]
[INFO] Operate Importer Common ............................ SUCCESS [  0.115 s]
[INFO] Operate Importer 8.5 ............................... SUCCESS [  0.116 s]
[INFO] Operate Importer 8.6 ............................... SUCCESS [  0.090 s]
[INFO] Operate Importer ................................... SUCCESS [  0.111 s]
[INFO] Operate Archiver ................................... SUCCESS [  0.208 s]
[INFO] Operate Webjar ..................................... SUCCESS [  0.089 s]
[INFO] Operate MVC Auth0 Commons (Jakarta) ................ SUCCESS [  0.178 s]
[INFO] Operate Webapp ..................................... SUCCESS [  0.088 s]
[INFO] Identity Parent .................................... SUCCESS [  0.119 s]
[INFO] Identity Common .................................... SUCCESS [  0.130 s]
[INFO] Camunda Authentication ............................. SUCCESS [  0.144 s]
[INFO] Identity Backend ................................... SUCCESS [  0.093 s]
[INFO] Identity Webjar .................................... SUCCESS [  0.078 s]
[INFO] Operate Data Generator ............................. SUCCESS [  0.208 s]
[INFO] Tasklist Parent .................................... SUCCESS [  0.070 s]
[INFO] Tasklist Common .................................... SUCCESS [  0.295 s]
[INFO] Tasklist ELS schema ................................ SUCCESS [  0.104 s]
[INFO] Tasklist Importer Common ........................... SUCCESS [  0.101 s]
[INFO] Tasklist Importer 850 .............................. SUCCESS [  0.102 s]
[INFO] Tasklist Importer 860 .............................. SUCCESS [  0.105 s]
[INFO] Tasklist Importer .................................. SUCCESS [  0.125 s]
[INFO] Tasklist Archiver .................................. SUCCESS [  0.109 s]
[INFO] Tasklist Webjar .................................... SUCCESS [  0.097 s]
[INFO] Tasklist MVC Auth0 Commons (Jakarta) ............... SUCCESS [  0.171 s]
[INFO] Tasklist Webapp .................................... SUCCESS [  0.196 s]
[INFO] Tasklist Data Generator ............................ SUCCESS [  0.107 s]
[INFO] Zeebe Distribution ................................. SUCCESS [  0.226 s]
[INFO] Camunda Spring SDK ................................. SUCCESS [  0.109 s]
[INFO] zeebe-client-java .................................. SUCCESS [  0.101 s]
[INFO] Zeebe Benchmark .................................... SUCCESS [  0.070 s]
[INFO] Zeebe QA ........................................... SUCCESS [  0.070 s]
[INFO] Zeebe QA Utilities ................................. SUCCESS [  0.137 s]
[INFO] Zeebe QA Integration Tests ......................... SUCCESS [  0.086 s]
[INFO] Zeebe QA Update Tests .............................. SUCCESS [  0.061 s]
[INFO] Zeebe Root ......................................... SUCCESS [  0.132 s]
[INFO] Operate QA ......................................... SUCCESS [  0.089 s]
[INFO] Operate QA Util .................................... SUCCESS [  0.068 s]
[INFO] Operate QA Data Generator .......................... SUCCESS [  0.145 s]
[INFO] Operate QA Integration Tests ....................... SUCCESS [  0.081 s]
[INFO] Operate QA Query Performace Tests .................. SUCCESS [  0.076 s]
[INFO] Operate QA Migration Tests Parent .................. SUCCESS [  0.059 s]
[INFO] Operate QA Migration Tests - Test Fixture 1.1.0 .... SUCCESS [  0.159 s]
[INFO] Operate QA Migration Tests - Test Fixture 1.2.0 .... SUCCESS [  0.195 s]
[INFO] Operate QA Migration Tests - Test Fixture 1.3.0 .... SUCCESS [  0.099 s]
[INFO] Operate QA Migration Tests - Test Fixture 8.0.0 .... SUCCESS [  0.101 s]
[INFO] Operate QA Migration Tests - Test Fixture 8.1.0 .... SUCCESS [  0.196 s]
[INFO] Operate QA Migration Tests - Test Fixture 8.2.0 .... SUCCESS [  0.100 s]
[INFO] Operate QA Migration Tests - Test Fixture 8.3.0 .... SUCCESS [  0.102 s]
[INFO] Operate QA Migration Tests - Test Fixture 8.4.0 .... SUCCESS [  0.197 s]
[INFO] Operate QA Migration Tests - Test Fixture 8.5.0 .... SUCCESS [  0.195 s]
[INFO] Operate QA Migration Tests ......................... SUCCESS [  0.051 s]
[INFO] Operate QA Migration Performance Test .............. SUCCESS [  0.113 s]
[INFO] Operate QA Backup Tests ............................ SUCCESS [  0.071 s]
[INFO] Tasklist QA ........................................ SUCCESS [  0.075 s]
[INFO] Tasklist QA Util ................................... SUCCESS [  0.253 s]
[INFO] Tasklist Integration Tests ......................... SUCCESS [  0.093 s]
[INFO] Tasklist QA Migration Tests Parent ................. SUCCESS [  0.088 s]
[INFO] Tasklist QA Migration Tests - Test Fixture 8.1.0 ... SUCCESS [  0.132 s]
[INFO] Tasklist QA Migration Tests - Test Fixture 8.2.0 ... SUCCESS [  0.133 s]
[INFO] Tasklist QA Migration Tests - Test Fixture 8.3.0 ... SUCCESS [  0.125 s]
[INFO] Tasklist QA Migration Tests - Test Fixture 8.4.0 ... SUCCESS [  0.130 s]
[INFO] Tasklist QA Migration Tests - Test Fixture 8.5.0 ... SUCCESS [  0.134 s]
[INFO] Tasklist QA Migration Tests ........................ SUCCESS [  0.080 s]
[INFO] Tasklist QA Backup Tests ........................... SUCCESS [  0.046 s]
[INFO] Tasklist aggregated coverage ....................... SUCCESS [  0.049 s]
[INFO] Camunda Process Test Java .......................... SUCCESS [  0.065 s]
[INFO] Camunda 8 Root ..................................... SUCCESS [  0.137 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  2.318 s (Wall Clock)
[INFO] Finished at: 2024-06-26T09:41:24+02:00
[INFO] ------------------------------------------------------------------------

```

With skipQaBuild

```
 mvnd validate -D skipQaBuild

[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Camunda 8 Root 8.6.0-SNAPSHOT:
[INFO] 
[INFO] Zeebe BOM .......................................... SUCCESS [  0.001 s]
[INFO] Zeebe Build Tools .................................. SUCCESS [  0.000 s]
[INFO] Zeebe Parent ....................................... SUCCESS [  0.039 s]
[INFO] Zeebe Util ......................................... SUCCESS [  0.113 s]
[INFO] Camunda Search Parent .............................. SUCCESS [  0.042 s]
[INFO] Camunda Search Client .............................. SUCCESS [  0.074 s]
[INFO] Camunda Search Client - Elasticsearch .............. SUCCESS [  0.090 s]
[INFO] Camunda Search Client - Opensearch ................. SUCCESS [  0.083 s]
[INFO] Camunda Search Client - Connect .................... SUCCESS [  0.116 s]
[INFO] Camunda Service .................................... SUCCESS [  0.069 s]
[INFO] Zeebe Atomix Parent Pom ............................ SUCCESS [  0.069 s]
[INFO] Zeebe Atomix Utilities ............................. SUCCESS [  0.093 s]
[INFO] Zeebe Scheduler .................................... SUCCESS [  0.083 s]
[INFO] Zeebe Protocol ..................................... SUCCESS [  0.197 s]
[INFO] Zeebe Exporter API ................................. SUCCESS [  0.071 s]
[INFO] Zeebe BPMN model API ............................... SUCCESS [  0.132 s]
[INFO] Zeebe Gateway Protocol ............................. SUCCESS [  0.041 s]
[INFO] Zeebe Gateway Protocol Implementation .............. SUCCESS [  0.111 s]
[INFO] Camunda Client Java ................................ SUCCESS [  0.211 s]
[INFO] Zeebe Test Util .................................... SUCCESS [  0.147 s]
[INFO] Zeebe Snapshots .................................... SUCCESS [  0.100 s]
[INFO] Zeebe Journal ...................................... SUCCESS [  0.100 s]
[INFO] Zeebe Atomix Cluster ............................... SUCCESS [  0.107 s]
[INFO] Zeebe DB ........................................... SUCCESS [  0.054 s]
[INFO] Zeebe Msgpack Core ................................. SUCCESS [  0.072 s]
[INFO] Zeebe Msgpack Value ................................ SUCCESS [  0.147 s]
[INFO] Zeebe Auth ......................................... SUCCESS [  0.078 s]
[INFO] Zeebe Protocol Implementation ...................... SUCCESS [  0.043 s]
[INFO] Zeebe Logstreams ................................... SUCCESS [  0.037 s]
[INFO] Zeebe Stream Platform .............................. SUCCESS [  0.043 s]
[INFO] Zeebe Dynamic Cluster Configuration ................ SUCCESS [  0.150 s]
[INFO] Zeebe Transport .................................... SUCCESS [  0.045 s]
[INFO] Zeebe Protocol AssertJ Assertions .................. SUCCESS [  0.071 s]
[INFO] Zeebe Protocol Test Util ........................... SUCCESS [  0.043 s]
[INFO] Zeebe Broker Client ................................ SUCCESS [  0.100 s]
[INFO] Zeebe Gateway ...................................... SUCCESS [  0.181 s]
[INFO] Zeebe Gateway gRPC ................................. SUCCESS [  0.270 s]
[INFO] Zeebe FEEL Integration ............................. SUCCESS [  0.057 s]
[INFO] Zeebe Expression Language .......................... SUCCESS [  0.149 s]
[INFO] Zeebe DMN Engine ................................... SUCCESS [  0.053 s]
[INFO] Zeebe Workflow Engine .............................. SUCCESS [  0.163 s]
[INFO] Zeebe Backup ....................................... SUCCESS [  0.084 s]
[INFO] Zeebe Backup Store Testkit ......................... SUCCESS [  0.095 s]
[INFO] Zeebe Backup Store for S3 .......................... SUCCESS [  0.386 s]
[INFO] Zeebe Backup Store Common .......................... SUCCESS [  0.306 s]
[INFO] Zeebe Backup Store for GCS ......................... SUCCESS [  0.264 s]
[INFO] Zeebe Backup Store for Azure ....................... SUCCESS [  0.230 s]
[INFO] Zeebe Exporter Test Harness ........................ SUCCESS [  0.051 s]
[INFO] Zeebe Broker ....................................... SUCCESS [  0.129 s]
[INFO] Zeebe Backup Restore ............................... SUCCESS [  0.041 s]
[INFO] Zeebe Gateway REST API server ...................... SUCCESS [  0.285 s]
[INFO] Zeebe Protocol Jackson ............................. SUCCESS [  0.036 s]
[INFO] Zeebe Elasticsearch Exporter ....................... SUCCESS [  0.082 s]
[INFO] Zeebe Opensearch Exporter .......................... SUCCESS [  0.089 s]
[INFO] Webapps Common ..................................... SUCCESS [  0.078 s]
[INFO] Operate Parent ..................................... SUCCESS [  0.063 s]
[INFO] Operate Common ..................................... SUCCESS [  0.139 s]
[INFO] Operate schema ..................................... SUCCESS [  0.105 s]
[INFO] Operate Importer Common ............................ SUCCESS [  0.072 s]
[INFO] Operate Importer 8.5 ............................... SUCCESS [  0.345 s]
[INFO] Operate Importer 8.6 ............................... SUCCESS [  0.101 s]
[INFO] Operate Importer ................................... SUCCESS [  0.276 s]
[INFO] Operate Archiver ................................... SUCCESS [  0.073 s]
[INFO] Operate Webjar ..................................... SUCCESS [  0.068 s]
[INFO] Operate MVC Auth0 Commons (Jakarta) ................ SUCCESS [  0.174 s]
[INFO] Operate Webapp ..................................... SUCCESS [  0.131 s]
[INFO] Identity Parent .................................... SUCCESS [  0.073 s]
[INFO] Identity Common .................................... SUCCESS [  0.092 s]
[INFO] Camunda Authentication ............................. SUCCESS [  0.075 s]
[INFO] Identity Backend ................................... SUCCESS [  0.147 s]
[INFO] Identity Webjar .................................... SUCCESS [  0.062 s]
[INFO] Operate Data Generator ............................. SUCCESS [  0.152 s]
[INFO] Tasklist Parent .................................... SUCCESS [  0.062 s]
[INFO] Tasklist Common .................................... SUCCESS [  0.213 s]
[INFO] Tasklist ELS schema ................................ SUCCESS [  0.080 s]
[INFO] Tasklist Importer Common ........................... SUCCESS [  0.059 s]
[INFO] Tasklist Importer 850 .............................. SUCCESS [  0.051 s]
[INFO] Tasklist Importer 860 .............................. SUCCESS [  0.157 s]
[INFO] Tasklist Importer .................................. SUCCESS [  0.142 s]
[INFO] Tasklist Archiver .................................. SUCCESS [  0.055 s]
[INFO] Tasklist Webjar .................................... SUCCESS [  0.062 s]
[INFO] Tasklist MVC Auth0 Commons (Jakarta) ............... SUCCESS [  0.106 s]
[INFO] Tasklist Webapp .................................... SUCCESS [  0.486 s]
[INFO] Tasklist Data Generator ............................ SUCCESS [  0.057 s]
[INFO] Zeebe Distribution ................................. SUCCESS [  0.111 s]
[INFO] Camunda Spring SDK ................................. SUCCESS [  0.126 s]
[INFO] zeebe-client-java .................................. SUCCESS [  0.039 s]
[INFO] Zeebe Benchmark .................................... SUCCESS [  0.071 s]
[INFO] Zeebe Root ......................................... SUCCESS [  0.043 s]
[INFO] Camunda Process Test Java .......................... SUCCESS [  0.067 s]
[INFO] Camunda 8 Root ..................................... SUCCESS [  0.038 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  2.072 s (Wall Clock)
[INFO] Finished at: 2024-06-26T09:42:03+02:00
[INFO] ------------------------------------------------------------------------
```

So yes it works with just specify -D skipQaBuild